charts/values.yaml

```diff
@@ -64,3 +79,277 @@ engines:
   replicated:
     mayastor:
       enabled: true
+
+loki:
+  enabled: true
+  # Configuration for loki's localpv hostpath storage class.
+  localpvScConfig:
+    enabled: true
+    loki:
+      # Name of loki's localpv hostpath storage class.
+      name: "openebs-loki-localpv"
+      # -- Host path where local loki data is stored in.
+      basePath: "/var/local/{{ .Release.Name }}/localpv-hostpath/loki"
+      # -- ReclaimPolicy of loki's localpv hostpath storage class.
+      reclaimPolicy: Delete
+      # -- VolumeBindingMode of loki's localpv hostpath storage class.
+      volumeBindingMode: WaitForFirstConsumer
+    minio:
+      # Name of minio's localpv hostpath storage class.
+      name: "openebs-minio-localpv"
+      # -- Host path where local minio data is stored in.
+      basePath: "/var/local/{{ .Release.Name }}/localpv-hostpath/minio"
+      # -- ReclaimPolicy of minio's localpv hostpath storage class.
+      reclaimPolicy: Delete
+      # -- VolumeBindingMode of minio's localpv hostpath storage class.
+      volumeBindingMode: WaitForFirstConsumer
+
+  loki:
+    schemaConfig:
+      configs:
+        - from: 2024-04-01
+          store: tsdb
+          object_store: s3
+          schema: v13
+          index:
+            prefix: loki_index_
+            period: 24h
+    commonConfig:
+      replication_factor: 3
+
+    # Configure these if you don't want to use minio and rather an external s3 bucket.
+    # storage:
+    #   type: s3
+    #   bucketNames:
+    #     chunks: <mybucket>
+    #   s3:
+    #     s3: s3://<my access id>:<my secret key>@<my endpoint>/<mybucket>
+    #     endpoint: <my endpoint>
+    #     region: <my region>
+    #     secretAccessKey: <my secret key>
+    #     accessKeyId: <my access id>
+    #   object_store:
+    #     type: s3
+    #     s3:
+    #       endpoint: s3://<my access id>:<my secret key>@<my endpoint>/<mybucket>
+    #       region: <my region>
+    #       access_key_id: <my access id>
+    #       secret_access_key: <my secret key>
+
+    ingester:
+      chunk_encoding: snappy
+      # Configure these if a quicker ingestion is needed, i.e. faster push to your bucket.
+      # chunk_idle_period: 3m
+      # chunk_retain_period: 1m
+      # max_chunk_age: 6m
+    tracing:
+      enabled: true
+    querier:
+      max_concurrent: 1
+    limits_config:
+      ingestion_burst_size_mb: 1000
+      ingestion_rate_mb: 10000
+
+  singleBinary:
+    replicas: 3
+    drivesPerNode: 1
+    persistence:
+      enabled: true
+      storageClass: "openebs-loki-localpv"
+      accessModes:
+        - ReadWriteOnce
+      size: 2Gi
+
+  minio:
+    replicas: 3
+    drivesPerNode: 1
+    mode: distributed
+    # Disable this if you want to enabled external s3 bucket, and uncomment the storage section above.
+    enabled: true
+    persistence:
+      enabled: true
+      storageClass: "openebs-minio-localpv"
+      size: 2Gi
+
+  deploymentMode: SingleBinary
+
+  lokiCanary:
```

Comment:
Do we need to explicitly disable all this? It's quite verbose

Reply:
Yes we need to other wise it will install a lot of components which we don't need.

Reply:
fair enough then :(